### DISCUSSION TITLE
Filament Upgrade problems ErrorException

### DISCUSSION THREAD

**Discussion by fagnerdireito** [filamentphp/filament#14140] [General]
When i execute 

`php artisan filament:upgrade`

 ```
ErrorException 

  copy(app.js): Failed to open stream: No such file or directory


```

![image](https://github.com/user-attachments/assets/9ac6ff20-3ccb-410e-abe6-e9ba4ddf6d57)

Environment
Laravel Version . 10.48.20  
PHP Version ........ 8.2.0  

Filament .................  
Packages ................. filament, forms, notifications, support, tables  
Version ......... v3.2.110  

Livewire .................  
Livewire .......... v3.5.6

**Comment by leandrocfe** [comment]
Did you customize something?
